src/main/java/com/velox/sloan/cmo/workflows/IgoLimsPluginUtils/Tags.java

```diff
@@ -0,0 +1,17 @@
+package com.velox.sloan.cmo.workflows.IgoLimsPluginUtils;
+
+public interface Tags {
```

Comment:
Should the interface name be more specific?

Reply:
Tags is the term I use for LIMS tags and Sapio also uses the same term, so I am staying consistent. Anyone working on LIMS would know what it means. Let me know if you still suggest to change the name.